protocol/src/main/java/org/jboss/as/protocol/mgmt/ActiveOperationImpl.java

```diff
@@ -92,7 +93,8 @@ public boolean done(T result) {
             }
 
             @Override
-            public boolean failed(Throwable t) {
+            public boolean failed(final Throwable t) {
```

Comment:
Could you add a Java doc describing the argument cannot be null?

Reply:
Fixed.